Sources/Afterpay/Model/Currency.swift

```diff
@@ -21,21 +16,32 @@ struct Currency {
 
   var symbol: String {
     switch code {
-    case australianLocale.currencyCode:
+    case Locales.australia.currencyCode:
       return "A$"
-    case newZealandLocale.currencyCode:
+    case Locales.newZealand.currencyCode:
       return "NZ$"
-    case canadianLocale.currencyCode:
+    case Locales.canada.currencyCode:
       return "CA$"
-    case unitedStatesLocale.currencyCode:
-      fallthrough // swiftlint:disable:this no_fallthrough_only
     default:
       return "$"
     }
   }
 
+  var locale: Locale {
+    switch code {
+    case Locales.australia.currencyCode:
+      return Locales.australia
+    case Locales.newZealand.currencyCode:
+      return Locales.newZealand
+    case Locales.canada.currencyCode:
+      return Locales.canada
+    default:
+      return Locales.unitedStates
+    }
+  }
```

Comment:
Is this being used anywhere?

Reply:
Yep: https://github.com/afterpay/sdk-ios/pull/112/files#diff-45a1a875c8c49852618515fed790702cR98

Reply:
```swift
public func setConfiguration(_ configuration: Configuration?) {
  Afterpay.configuration = configuration

  // Currently the locale for terms of use is strongly tied to the purchasing currency
  // this will most likely be separated in the future
  Afterpay.locale = configuration?.currency.locale ?? Locales.unitedStates

  notificationCenter.post(name: .configurationUpdated, object: configuration)
}
```

Reply:
Ah, I missed that. Sweet 👌